components/raftstore/src/store/fsm/peer.rs

```diff
@@ -133,6 +133,9 @@ enum DelayReason {
 /// in most case.
 const MAX_REGIONS_IN_ERROR: usize = 10;
 const REGION_SPLIT_SKIP_MAX_COUNT: usize = 3;
+/// Limits the request size that can be batched in a single RaftCmdRequest.
+// todo: this fugure maybe changed to a more suitable value.
+const MAX_BATCH_SIZE_LIMIT: u64 = 1000 * 1000;
```

Comment:
Since `max_grpc_send_msg_len` has limited the max length of grpc messages, how about reusing it as the limit of BatchCmdRequest.
https://github.com/tikv/tikv/blob/af3903947ff1db0d1436944016878097c9d8bac8/src/server/config.rs#L70

Reply:
I think they have different semantics.

Reply:
```suggestion
const DEFAULT_MAX_BATCH_SIZE_LIMIT: u64 = 10 * 1024 * 1024;
```

Reply:
why 10MB?

Reply:
lapse, pls correct it with 1MB == 1 * 1024 * 1024